GAE/src/org/waterforpeople/mapping/dataexport/RawDataSpreadsheetImporter.java

```diff
@@ -307,18 +310,50 @@ public InstanceData parseInstance(Sheet sheet, int startRow,
                 Cell cell = iterationRow.getCell(columnIndex);
 
                 if (cell != null) {
-                    if (isGeoQuestion) {
-                        String latitude = ExportImportUtils.parseCellAsString(cell);
-                        String longitude = ExportImportUtils.parseCellAsString(iterationRow
-                                .getCell(columnIndex + 1));
-                        String elevation = ExportImportUtils.parseCellAsString(iterationRow
-                                .getCell(columnIndex + 2));
-                        String geoCode = ExportImportUtils.parseCellAsString(iterationRow
-                                .getCell(columnIndex + 3));
-                        val = latitude + "|" + longitude + "|" + elevation + "|" + geoCode;
-                    } else {
-                        val = ExportImportUtils.parseCellAsString(cell);
+                    switch (questionType) {
+                        case GEO:
+                            String latitude = ExportImportUtils.parseCellAsString(cell);
+                            String longitude = ExportImportUtils.parseCellAsString(iterationRow
+                                    .getCell(columnIndex + 1));
+                            String elevation = ExportImportUtils.parseCellAsString(iterationRow
+                                    .getCell(columnIndex + 2));
+                            String geoCode = ExportImportUtils.parseCellAsString(iterationRow
+                                    .getCell(columnIndex + 3));
+                            val = latitude + "|" + longitude + "|" + elevation + "|" + geoCode;
+                            break;
+                        case CASCADE:
+                            // Two different possible formats:
+                            // With codes: code1:val1|code2:val2|...
+                            // Without codes: val1|val2|...
+                            String cascadeString = ExportImportUtils.parseCellAsString(cell);
+                            String[] cascadeParts = cascadeString.split("\\|");
+                            List<Map<String, String>> cascadeList = new ArrayList<>();
+                            for (String cascadeNode : cascadeParts) {
+                                String[] codeAndName = cascadeNode.split(":");
```

Comment:
just to be absolutely :) sure we should also use `split(":", 2)` here. so we don't get something like `code:`